lib/points.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class PointsRender {
+  void render(Canvas canvas, int points, Size screenSize) {
+    final width = ((screenSize.width / 2) - 30);
+    final position = Offset(width, 100);
+
+    final text = TextPainter(
```

Comment:
Flame already has a nice class to handle this, it is called TextConfig https://flame-engine.org/docs/text.md

Reply:
Thanks! It's more simple than i was doing